PracticeProjectTests/Mock/MockUserAPI.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MockUserAPI.swift
+//  PracticeProjectTests
+//
+//  Created by Gnanavel on 01/04/22.
+//
+
+import Foundation
+@testable import PracticeProject
+
+/// MockUserAPI is mock api for testing the user api
+class MockUserAPI: API {
```

Comment:
Can you list the purposes of this class in bulletins? (Short sentences of what it achieves)

Reply:
* I want to test the ViewModel by using this mock API class.
* If I want to test my unit test cases I need some hard-coded values that's why I have created.